src/ui/zcl_abapgit_gui_page_repo_over.clas.abap

```diff
@@ -446,6 +443,29 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_REPO_OVER IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD shorten_repo_url.
+    FIELD-SYMBOLS <ls_match_result> TYPE match_result.
+    DATA lt_results TYPE match_result_tab.
+    DATA lv_new_length TYPE i.
+
+    rv_shortened = iv_full_url.
+
+    REPLACE FIRST OCCURRENCE OF 'https://' IN rv_shortened WITH ''.
+    REPLACE FIRST OCCURRENCE OF 'http://' IN rv_shortened WITH ''.
+    IF rv_shortened CP '*.git'.
+      lv_new_length = strlen( rv_shortened ) - 4.
+      rv_shortened  = rv_shortened(lv_new_length).
+    ENDIF.
+    FIND ALL OCCURRENCES OF '/' IN rv_shortened RESULTS lt_results.
+    IF sy-subrc = 0 AND lines( lt_results ) >= 2.
+      " host.domain/org_or_user/repo/...
+      READ TABLE lt_results INDEX 2 ASSIGNING <ls_match_result>.
```

Comment:
Needs to cut at 3, otherwise we only see the gitHub user/organization name.
Several of my repos now only show URL of `github.com/pokrakam`

Reply:
This was on purpose, so it shortens the column but you're still aware if it's switched to a fork, 

`github.com/abapgit/abapgit` and `github.com/frehu/abapgit` have the common part `abapgit`, so I wanted to cut that. Otherwise there is nothing to shorten as `https` and `.git` is already removed.

but I'm going to go with @mbtools' suggestion, I think this is the best variant yet